etc/dovetail-edge-deploy.yml

```diff
@@ -0,0 +1,865 @@
+      StitchLambdaFunctionArn:
+        default: Unqualified Stitch Lambda function ARN
+Parameters:
+  StagingOriginBucket:
+    Default: prx-dovetail.s3.amazonaws.com
+    Description: >
+      The DNS name of the Amazon S3 bucket from which you want CloudFront to
+      get objects for this origin, for example, myawsbucket.s3.amazonaws.com.
+    Type: String
+  StagingOriginPath:
+    Default: /stitch-staging
+    Description: >
+      An optional element that causes CloudFront to request your content from a
+      directory in your Amazon S3 bucket or your custom origin, beginning with
+      a / (forward slash).
+    Type: String
+  StagingDistributionDomain:
+    Default: dovetail-cdn-staging.prxu.org
+    Description: eg. dovetail-cdn.prxu.org
+    Type: String
+  ProductionOriginBucket:
+    Default: prx-dovetail.s3.amazonaws.com
+    Description: >
+      The DNS name of the Amazon S3 bucket from which you want CloudFront to
+      get objects for this origin, for example, myawsbucket.s3.amazonaws.com.
+    Type: String
+  ProductionOriginPath:
+    Default: /stitch-production
+    Description: >
+      An optional element that causes CloudFront to request your content from a
+      directory in your Amazon S3 bucket or your custom origin, beginning with
+      a / (forward slash).
+    Type: String
+  ProductionDistributionDomain:
+    Default: dovetail-cdn.prxu.org
+    Description: eg. dovetail-cdn.prxu.org
+    Type: String
+  InfrastructureNotificationsStackName:
+    Default: infrastructure-notifications
+    Type: String
+  GitHubToken:
+    Description: >
+      A GitHub access token with permissions to update statuses, request
+      content, etc
+    Type: String
+  GitHubWebhookSecret:
+    Description: >
+      A secret used to sign requests made by GitHub to webhook endpoints.
+      This secret is shared by all webhooks created by this template.
+    Type: String
+  StagingCloudFormationStackName:
+    Default: cloudfront-dovetail-cdn-staging
+    Description: >
+      The name of the CloudFormation stack that manages the staging deployment
+      of Dovetail CloudFront distribution
+    Type: String
+  ProductionCloudFormationStackName:
+    Default: cloudfront-dovetail-cdn-production
+    Description: >
+      The name of the CloudFormation stack that manages the production
+      deployment of Dovetail CloudFront distribution
+    Type: String
+  StagingStitchLambdaFunctionArn:
+    Description: >
+      e.g., arn:aws:lambda:<region>:<account>:function:<name>
+    Type: String
+  StagingBytesLambdaFunctionArn:
+    Description: >
+      e.g., arn:aws:lambda:<region>:<account>:function:<name>
+    Type: String
+  ProductionStitchLambdaFunctionArn:
+    Description: >
+      e.g., arn:aws:lambda:<region>:<account>:function:<name>
+    Type: String
+  ProductionBytesLambdaFunctionArn:
+    Description: >
+      e.g., arn:aws:lambda:<region>:<account>:function:<name>
+    Type: String
+  CertificateArn:
```

Comment:
Do dovetail staging/prod share a cert?

Reply:
I think they used to, and in April we added a cert to dovetail-cdn.yml that it manages on its own, so this should be removed.